### PR TITLE
Add pytspack

### PR DESCRIPTION
**Pull request by bbakernoaa**
This package is for a python interface to Robert J. Renka's fortran TSPACK: Tension Spline Curve Fitting Package

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pytspack/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  sha256: 3e35e2c56ace4452dc3aadf030f081581da7257044cd66f24c3343f396cdca1e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - python >=3.6
```
Comment (marcelotrevisani) [inline recipes/pytspack/meta.yaml:19]: I think ``python`` should not be here, it should be just in ``host`` and ``run``
```suggestion
```
What was your idea for adding it here?

Comment (bbakernoaa) [reply recipes/pytspack/meta.yaml]: removed

File: recipes/monet/meta.yaml [hunk 2]
```diff
@@ -1,60 +0,0 @@
-{% set name = "monet" %}
-{% set version = "2.1.5" %}
```
Comment (marcelotrevisani) [inline recipes/monet/meta.yaml:2]: could you please revert it?
Please do not delete the folders on ``recipes``
Because we may face some problems

Comment (bbakernoaa) [reply recipes/monet/meta.yaml:2]: This was to another recipe that I had.  It is not part of the standard conda-forge/staged-recipes

File: recipes/pytspack/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
+    - numpy
```
Comment (marcelotrevisani) [inline recipes/pytspack/meta.yaml:25]: are you compiling agains ``numpy``? if you are, it should be
```suggestion
    - {{ pin_compatible('numpy') }}
```

Comment (bbakernoaa) [reply recipes/pytspack/meta.yaml:25]: This requires numpy to be installed as it uses f2py to compile the fortran code using numpy.distutil.  Is there a better way to do this?

Comment (bbakernoaa) [reply recipes/pytspack/meta.yaml]: fixed

**Comment by bbakernoaa** [comment]
What is the next step for this?

**Comment by bbakernoaa** [comment]
@marcelotrevisani can this proceed?

**Comment by marcelotrevisani** [comment]
Sorry for the delay, but could you please revert the file which you deleted?

**Pull request closed without merging**